contracts/stellar-axelar-gas-service/src/interface.rs

```diff
@@ -8,10 +8,27 @@ pub trait AxelarGasServiceInterface {
     /// Pay for gas using a token for sending a message on a destination chain.
     ///
     /// This function is called on the source chain before calling the gateway to send a message.
-    ///
-    /// `sender` refers to the address that sent the cross-chain message via the `axelar_gateway`.
     /// The `spender` pays the gas but might differ from the `sender`,
     /// e.g. the `sender` is a contract, but the `spender` can be the user signing the transaction.
+    ///
+    /// # Arguments
+    /// * `sender` - The address initiating the gas payment. It's the address that sent the cross-chain message via the `axelar_gateway`.
+    /// * `destination_chain` - The name of the destination chain where the transaction will be executed.
```

Comment:
```suggestion
    /// * `destination_chain` - The destination chain for the message.
```
A bit more concise, and consistent with usage of message terminology